search-commons/src/test/java/no/unit/nva/search2/ResourceClientTest.java

```diff
@@ -169,7 +168,7 @@ void shouldCheckMapping() {
         @Test
         void shouldCheckFacets() throws BadRequestException {
```

Comment:
used this test to verify that splitting in to queries and merging the results where successful (by debugging)
Since every test also were successful, I expected this to work flawlessly, but wanted to see it in action....